src/account/EditProfile.html

```diff
@@ -161,9 +161,7 @@ <h2 style="margin-bottom: 20px;">{ __("account / delete / hed") }</h2>
         {:elseif deleteAccount}
         <h3><i class="fa fa-times"></i> { __("account / confirm-account-deletion") }</h3>
         <button on:click="set({deleteAccount: false})" class="btn btn-back btn-primary">
-            <!-- prettier-ignore -->
-            <i class="fa fa-chevron-left"></i> &nbsp; { __("account / confirm-account-deletion /
-            no") }
+            // prettier-ignore <i class="fa fa-chevron-left"></i> &nbsp; { __("account / confirm-account-deletion / no") }
```

Comment:
this is nonsense though

Reply:
I did it because I saw the same in the same file [here](https://github.com/datawrapper/datawrapper/blob/master/src/account/EditProfile.html#L102)

and the previous `<!-- prettier-ignore -->` wasn't working and that line kept getting line broken so I assumed it was something special. i wanted to run prettier to check but it changed a bunch of files so instead i just went on a limb

Reply:
yeah then that is wrong too! `// prettier-ignore` will just be displayed in the UI. and i agree that the code formatter is dumb in these cases

Reply:
i mean i thought it was weird but I assumed there'd be a reason. But the actual `<!-- prettier-ignore -->` wasn't working, and the line kept getting formatted, and the fact that i committed it without the line break is the reason that tests are failling

Reply:
so why isn't `<!--prettier-ignore -->` working?

Reply:
no idea, but you can just split up the line to make it shorter to make it shut up